### ISSUE TITLE
The MMScan evaluation

### ISSUE THREAD

**Issue by SiyuanWei** [ZCMax/LLaVA-3D#9]
I noticed that mmscan authors has not released the entire dataset but a beta-version in 2024.9. Is the result in LLAVA-3d on MMScan benchmark evaluated on a partial version like beta-version, or do you have the entire version but it's private ?

**Comment by ZCMax** [comment]
Yes, I  have the entire version but it's private now, the full version would be released at the end of the last month/beginning of the next month.
